plugins/grafana-custom-plugins/grafana-chord-plugin/.config/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG grafana_version=latest
```

Comment:
should we use latest version or verified version? (I don't think this file matters that much to be honest, but maybe we could match it with the grafana version we use in our deployment as well?)

Reply:
I've removed this file as we don't need it. Thanks!

Reply:
I thought the files under `./config` are all automatically generated. How do we distinguish which files are needed and which are not?

Reply:
Do we need Dockerfile to build the image for grafana? I think this file is redundant even it is auto-generated?